defaults/main.yml

```diff
@@ -39,6 +39,7 @@ openstack_glance_demo_pass: glance_pass_default
 
 # Database
 openstack_glance_database_url: sqlite:////var/lib/glance/glance.sqlite
+openstack_glance_database_schema_version: 44
```

Comment:
Yes, unfortunately I couldn't find any convenient way to get the target migration version for currently installed version of OpenStack package. Though, I don't suppose it's a big issue since :
* Target migration version should only change between major releases.
* We should at least check, and probably need to tweak something between releases.
* User can override the target migration version since it's a standard variable.

Though, I have to admit the name of variable is something that may be improved.